### PR TITLE
loader: update log

### PR DESCRIPTION
**Pull request by cwen0**
update the database or table already exists error log
@zhangjinpeng1987 @qiuyesuifeng

### CODE REVIEW

**Comment by cwen0** [comment]
@tiancaiamao  PTAL

File: loader/loader.go [hunk 1]
```diff
@@ -480,9 +478,8 @@ func (l *Loader) restoreData() error {
 		log.Infof("[loader][run db schema]%s[start]", dbFile)
 		err = l.restoreSchema(l.conns[0], dbFile, "")
 		if err != nil {
-			err = causeErr(err)
-			if e, ok := err.(*mysql.MySQLError); ok && e.Number == tidb.ErrDBCreateExists {
-				log.Warnf("run db schema failed - %v", errors.ErrorStack(err))
+			if isErrDbTableExists(err) {
+				log.Infof("[loader][database already exists, skip]%s", dbFile)
```
Comment (nolouch) [inline loader/loader.go]: You must contact the context to know the return error  relate to  table or database. seems not good .

**Comment by nolouch** [comment]
LGTM

File: loader/db.go [hunk 2]
```diff
@@ -18,9 +18,10 @@ import (
 	"fmt"
 	"time"
 
-	_ "github.com/go-sql-driver/mysql"
+	"github.com/go-sql-driver/mysql"
 	"github.com/juju/errors"
 	"github.com/ngaut/log"
+	tidb "github.com/pingcap/tidb/mysql"
```
Comment (qiuyesuifeng) [inline loader/db.go]: Using `tidb` as a alias name is not so clear, how about `tmysql`?
The same problem in `loader/loader.go`

File: loader/db.go [hunk 3]
```diff
@@ -83,7 +83,9 @@ func executeSQL(conn *Conn, sqls []string, enableRetry bool, skipConstraintCheck
 		return nil
 	}
 	if err != nil {
-		log.Errorf("exec sqls[%-.100v] failed %v", sqls, errors.ErrorStack(err))
+		if !(isErrDBExists(err) || isErrTableExists(err)) {
```
Comment (qiuyesuifeng) [inline loader/db.go]: Why did you add this condition?

Comment (cwen0) [inline loader/db.go]: When a database or table exists,  that is to recover from a checkpoin, so should not be an error log

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by iamxy** [comment]
LGTM
